### PR TITLE
reflect case-server changes

### PR DESCRIPTION
**Pull request by AbdelHedhili**
Signed-off-by: AbdelHedhili <abdelsalem.hedhili@rte-france.com>

**Please check if the PR fulfills these requirements** *(please use `'[x]'` to check the checkboxes, or submit the PR and then click the checkboxes)*
- [x] The commit message follows our guidelines
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** *(Bug fix, feature, docs update, ...)*
Update to reflect to changes in case server api

**What is the current behavior?** *(You can also link to an open issue here)*
not updated with case server new api

**What is the new behavior (if this is a feature change)?**
updated with case server new api

### CODE REVIEW

File: cgmes-gl-server/src/main/java/com/powsybl/cgmes/gl/server/services/CgmesGlService.java [hunk 1]
```diff
@@ -85,26 +84,16 @@ public void toGeoDataServer(String caseName, Set<Country> countries) {
...
+        CaseDataSourceClient caseServerDataSource = createCaseServerDataSource(caseUuid);
         CgmesImport importer = new CgmesImport();
         Properties properties = new Properties();
         properties.put("iidm.import.cgmes.post-processors", "cgmesGLImport");
-        return importer.importData(caseServerDataSource, properties);
-    }
-
-    CaseDataSourceClient createCaseServerDataSource(String caseName) {
-        return new CaseDataSourceClient(caseServerBaseUri, caseName);
+        return importer.importData(caseServerDataSource, NetworkFactory.findDefault(), properties);
```
Comment (ChamseddineBhd) [inline cgmes-gl-server/src/main/java/com/powsybl/cgmes/gl/server/services/CgmesGlService.java:92]: Why did you add  NetworkFactory.findDefault() ? isn't already the default behaviour ?

Comment (AbdelHedhili) [reply cgmes-gl-server/src/main/java/com/powsybl/cgmes/gl/server/services/CgmesGlService.java:92]: importData(datasource, properties) is now deprecated, that's why I changed it to this new version

**Comment by AbdelHedhili** [comment]
@ChamseddineBhd The build error in the CI is because this PR is based on a change I did in the case-server but it's not yet merged so it uses the old caseServerClient and fails. Will be fixed by its own when the case-server PR is merged.
